From bd3130105cc359cae22c4ed548d618de5a6a4424 Mon Sep 17 00:00:00 2001
From: Simon Horman <horms+renesas@verge.net.au>
Date: Wed, 3 Jul 2013 13:38:17 +0900
Subject: [PATCH 0935/1083] Revert "drm/rcar-du: Don't ignore
 rcar_du_crtc_create() return value"

This reverts commit 4cfa9f5e61766af0589d96ccbd2cb702b3d99362.
(cherry picked from commit da6dfce4f1db3c29290522e95eb3d9b40b8c810c)

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 drivers/gpu/drm/rcar-du/rcar_du_kms.c | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/drivers/gpu/drm/rcar-du/rcar_du_kms.c b/drivers/gpu/drm/rcar-du/rcar_du_kms.c
index fb31485..5ba802c 100644
--- a/drivers/gpu/drm/rcar-du/rcar_du_kms.c
+++ b/drivers/gpu/drm/rcar-du/rcar_du_kms.c
@@ -200,11 +200,8 @@ int rcar_du_modeset_init(struct rcar_du_device *rcdu)
 	if (ret < 0)
 		return ret;
 
-	for (i = 0; i < ARRAY_SIZE(rcdu->crtcs); ++i) {
-		ret = rcar_du_crtc_create(rcdu, i);
-		if (ret < 0)
-			return ret;
-	}
+	for (i = 0; i < ARRAY_SIZE(rcdu->crtcs); ++i)
+		rcar_du_crtc_create(rcdu, i);
 
 	rcdu->used_crtcs = 0;
 	rcdu->num_crtcs = i;
-- 
1.8.3.2

